logging/logging.go

```diff
@@ -967,65 +968,70 @@ func toLogEntryInternalImpl(e Entry, l *Logger, parent string, skipLevels int) (
 // entry represents the fields of a logging.Entry that can be parsed by Logging agent.
 // See the mappings at https://cloud.google.com/logging/docs/structured-logging#special-payload-fields
 type structuredLogEntry struct {
-	// JsonMessage    map[string]interface{}        `json:"message,omitempty"`
-	// TextMessage    string                        `json:"message,omitempty"`
-	Message        json.RawMessage               `json:"message"`
-	Severity       string                        `json:"severity,omitempty"`
-	HTTPRequest    *logtypepb.HttpRequest        `json:"httpRequest,omitempty"`
-	Timestamp      string                        `json:"timestamp,omitempty"`
-	Labels         map[string]string             `json:"logging.googleapis.com/labels,omitempty"`
-	InsertID       string                        `json:"logging.googleapis.com/insertId,omitempty"`
-	Operation      *logpb.LogEntryOperation      `json:"logging.googleapis.com/operation,omitempty"`
-	SourceLocation *logpb.LogEntrySourceLocation `json:"logging.googleapis.com/sourceLocation,omitempty"`
-	SpanID         string                        `json:"logging.googleapis.com/spanId,omitempty"`
-	Trace          string                        `json:"logging.googleapis.com/trace,omitempty"`
-	TraceSampled   bool                          `json:"logging.googleapis.com/trace_sampled,omitempty"`
+	Message        json.RawMessage                   `json:"message"`
+	Severity       string                            `json:"severity,omitempty"`
+	HTTPRequest    *structuredLogEntryHTTPRequest    `json:"httpRequest,omitempty"`
```

Comment:
Since datatype of HTTPRequest is being changed, this can be a breaking change if the user is able to access this field. I see that structuredLogEntry type itself is unexported. So, it is highly unlikely but please recheck the rest of the logging code to ensure the same.